### PR TITLE
Documentation revamp

### PR DESCRIPTION
**Pull request by elsandosgrande**

### CODE REVIEW

**Comment by InBetweenNames** [comment]
@elsandosgrande is this ready to be merged?

**Comment by elsandosgrande** [comment]
All right, links are working now. Go ahead.

**Comment by InBetweenNames** [comment]
Thanks for the contribution!

**Comment by elsandosgrande** [comment]
Sure thing.


Also, I am writing this through email as Firefox Nightly is currently 
unusable, since only a tiny square in the top left corner responds to 
the pointer. Firefox stable is not affected, so this should be resolved 
later today when a new build of Nightly lands.

File: CONTRIBUTING_bs-Cyril.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+Код стварања захтјева, наслов би требао бити:
+
+~~~ text
+<категорија>/<пакет>: <извршни сажетак>
+~~~
+
+Ако захтјев додаје нешто у `ltoworkarounds.conf`, побрините се да има коментар поред релевантних линија који објашњава проблем с којим сте се сусрели.
+
+Ако захтјев садржи закрпу која омогућава да се програм изгради са оптимизацијама из овог оверлеја, пожељно је да је закрпа корисничка закрпа и да се налази у `patches` директорији, постепено повећавајући број ревизије пакета `sys-config/lotize` по потреби.
```
Comment (Bigg29) [inline CONTRIBUTING_bs-Cyril.md:15]: CONTRIBUTING_bs-Cyril.md

**Review by Bigg29 — APPROVED** [review]

**Pull request merged**